setup.sh

```diff
@@ -47,53 +47,51 @@ function check_dependencies()
   local pip_package_list=''
   local cmd=''
   local distro
-
+  declare -a debian=("dpkg-query -s " "apt install -y")
```

Comment:
This still fails to tell us whether a package is installed or not. Take, for instance, `git`. Remove it if you already have it installed, then use the `setup.sh` script to install kw dependencies.
```
$ ./setup.sh -i
Checking dependencies ...
Can we install the following dependencies git-email ? [y/N]: 
```
Notice that the user is not prompted to install `git`.
If we run `dpkg-query -s git`, we have an output that produces 0 as an exit code even though the package is not installed:
```
$ dpkg-query -s git
Package: git
Status: deinstall ok config-files
```
This is why you have to adapt the command previously used to fit into this refactoring or find another way to query this.